index.ftd

```diff
@@ -1,51 +1,1617 @@
+for everyone.
+
+We enable teams to iterate quickly and develop, preview, and ship delightful
+user experiences. Fastn has zero-configuration support for 35+ frontend
+frameworks and integrates with your headless content, commerce, or database of
+choice.
+
+-- image:
+src: $assets.files.static.doc2.png
+width: fill-container
+
+-- code:
+lang: ftd
+
+\-- FTD.ROW
+
+npx create-docusaurus@latest my-website classic
+
+-- h2: Existing login connection
+id: existing-login-connection
+
+Our suite of products are open source and available on Github. You are free to
+download install and customize them to your needs.We’d love to hear your
+feedback and suggestions, and collectively make Documentation easier and better
+for everyone.
+
+We enable teams to iterate quickly and develop, preview, and ship delightful
+user experiences. Fastn has zero-configuration support for 35+ frontend
+frameworks and integrates with your headless content, commerce, or database of
+choice.
+
+-- code:
+lang: ftd
+
+\-- FTD.ROW
+
+npx create-docusaurus@latest my-website classic
+
+-- h2: Choosing a connection when creating a project
+id: choosing-a-connection-when-creating-a-project
+
+Our suite of products are open source and available on Github. You are free to
+download install and customize them to your needs.We’d love to hear your
+feedback and suggestions, and collectively make Documentation easier and better
+for everyone.
+
+We enable teams to iterate quickly and develop, preview, and ship delightful
+user experiences. Fastn has zero-configuration support for 35+ frontend
+frameworks and integrates with your headless content, commerce, or database of
+choice.
+
+-- image:
+src: $assets.files.static.doc3.png
+width: fill-container
+
+-- code:
+lang: ftd
+
+\-- FTD.ROW
+
+npx create-docusaurus@latest my-website classic
+
+-- h2: Known limitations
+id: know-limitations
+
+Our suite of products are open source and available on Github. You are free to
+download install and customize them to your needs.We’d love to hear your
+feedback and suggestions, and collectively make Documentation easier and better
+for everyone.
+
+We enable teams to iterate quickly and develop, preview, and ship delightful
+user experiences. Fastn has zero-configuration support for 35+ frontend
+frameworks and integrates with your headless content, commerce, or database of
+choice.
+
+-- end: page
+
+
+
+
+
+
+
+
+
+
+-- component page:
+optional caption title:
+optional string document-title:
+optional string document-description:
+optional ftd.raw-image-src document-image: 
+optional body body:
+optional ftd.image-src site-logo: 
+integer logo-width: $common.logo-width
+integer logo-height: $common.logo-height
+optional string $site-url: $common.site-url
+optional string site-name: $common.site-name
+pr.toc-item list toc: $common.sitemap.toc
+optional string login-btn-txt: Login
+optional string login-btn-url: index.html
+boolean login-button: false
```

Comment:
```suggestion
optional string login-btn-txt: Login
optional string login-btn-url: index.html
boolean login-button: false
```
@MeenuKumari28 these are not there in Set-1, take latest Set-1 pull and add it there and raise PR